### ISSUE TITLE
Removing of multiple objects with preserved object stacking

### ISSUE THREAD

**Issue by mrgrimmig** [kangax/fabric.js#4109]
Hello,
i want to delete all currently selected objects at once. Since i set the property `preserveObjectStacking` to true, it throws an error and deletes just one of the selected objects.  
One workaround is to set the flag to false before deleting and set to true again when finished (line 14 & line 19 in the fiddle).

## Version
1.7.16

## Test Case
https://jsfiddle.net/ak68wm19/1/

## Steps to reproduce
1. Open fiddle
2. Select more than one object
3. Press delete

## Expected Behavior
All objects should be deleted.

## Actual Behavior
Following error will be thrown: 
```
Uncaught TypeError: Cannot read property 'getActiveGroup' of undefined
    at i._renderControls (VM139 fabric.min.js:4)
    at i._renderControls (VM139 fabric.min.js:6)
    at i._renderControls (VM139 fabric.min.js:1)
    at i.drawControls (VM139 fabric.min.js:4)
    at i.renderCanvas (VM139 fabric.min.js:3)
    at i.renderAll (VM139 fabric.min.js:3)
    at i.remove (VM139 fabric.min.js:1)
    at i.remove (VM139 fabric.min.js:4)
    at selection.forEach.obj (VM138:61)
    at Array.forEach (<anonymous>)
```

**Comment by asturur** [comment]
i think that removing the objects of the activeGroup trigger some sort of deselection process.
Worth a check.

**Comment by asturur** [comment]
https://jsfiddle.net/ak68wm19/2/

 this can be fixed deleting first the activeGroup and then the objects.
I m working on a fix anyway.

**Comment by asturur** [comment]
closed with #4120
